### PR TITLE
Fix typo related to Manage APIs Configuration perm.

### PR DESCRIPTION
**Pull request by webbj74**

### CODE REVIEW

File: modules/ROOT/pages/building-https-proxy.adoc [hunk 1]
```diff
@@ -15,7 +15,7 @@ To configure your HTTPS API Proxy using secrets stored in your secret group, you
 * Secrets Manager Permissions enabled for your user. +
 To configure an existing TLS context for the API Proxy, you need the *Read secrets metadata* permission.
 Review the xref:anypoint-security::asm-permission-concept.adoc[Secrets Manager Permissions] reference for more information.
-* Make sure your user has *Manage API Configuration* role.
+* Make sure your user has the *Manage APIs Configuration* permission for the environment the proxy will be created in.
```
Comment (valkyrie69) [inline modules/ROOT/pages/building-https-proxy.adoc:18]:
```suggestion
* Make sure your user has the *Manage APIs Configuration* permission for the environment in which to create the proxy. 
```

File: modules/ROOT/pages/building-https-proxy.adoc [hunk 2]
```diff
@@ -15,7 +15,7 @@ To configure your HTTPS API Proxy using secrets stored in your secret group, you
...
 * A secret group along with a TLS context in Secrets Manager. +
```
Comment (valkyrie69) [inline modules/ROOT/pages/building-https-proxy.adoc:19]: Secrets manager is lc

File: modules/ROOT/pages/building-https-proxy.adoc [hunk 3]
```diff
@@ -15,7 +15,7 @@ To configure your HTTPS API Proxy using secrets stored in your secret group, you
...
 Follow the xref:anypoint-security::asm-secret-group-creation-task.adoc[Create a Secret Group] instructions along with the  xref:anypoint-security::asm-secret-group-creation-task.adoc#add-a-tls-context[Add a TLS Context] task contained in that article if you need help creating a TLS context. +
```
Comment (valkyrie69) [inline modules/ROOT/pages/building-https-proxy.adoc:20]: For adding a TLS context, the correct xref is:
xref:anypoint-security::secret-group-add-tls-context.adoc#add-tls-context[Add a TLS Context]

Comment (valkyrie69) [inline modules/ROOT/pages/building-https-proxy.adoc:20]:
```suggestion
Follow the xref:anypoint-security::asm-secret-group-creation-task.adoc[Create a Secret Group] instructions and xref:anypoint-security::secret-group-add-tls-context.adoc#add-tls-context[Add a TLS Context] instructions if you need help creating a TLS context. +
```

**Comment by glenn-rodgers-sf** [comment]
Closing as PR is no longer relevant.

**Pull request closed without merging**